### ISSUE TITLE
Reproducing numbers for WIKISQL dataset 

### ISSUE THREAD

**Issue by vishwajeetkumar93** [google-research/tapas#65]
Hi
Thank you for releasing the pre-trained models for wikisql. I am trying to test the pre-trained model tapas_wikisql_sqa_masklm_large.zip. Using default hyper-parameter values given in readme and colab notebook, I am very low (approx 3%) denotation accuracy. As I read in other issues: you have talked about weird biases, how can we get away with it and get numbers similar to what is reported in the paper for wikisql dataset.? Do i need to update some other hyper-parameters for running the code/model on wikisql dataset. Some help in this regard will be much appreciated.

**Comment by thomasmueller-google** [comment]
Thanks!

Can you share the commands you ran?

I understand you are evaluating the wikisql model on the wikisql data.
That should give you exactly the results we are reporting in the README (88.62% denotation accuracy).

**Comment by vishwajeetkumar93** [comment]
Hey it seems I had set task to WIKISQL. I replaced it with WIKISQL_SUPERVISED and it worked. Thanks

**Comment by aminfardi** [comment]
@vishwajeetkumar93, are you able to share your command? I'm trying to do the same, but getting very low accuracies. I'm using the WIKISQL as I'm looking for the weakly supervised performance on tapas_wikisql_sqa_masklm_small_reset. Unless I'm misunderstanding what WIKISQL does?

**Comment by vishwajeetkumar93** [comment]
Hi Amin
Please use WIKISQL_SUPERVISED as value for task hyperparameter.

**Comment by aminfardi** [comment]
@vishwajeetkumar93, I used WIKISQL_SUPERVISED  but [I'm still unable to reproduce](https://github.com/google-research/tapas/issues/75) the README numbers:

```
`tf.data.TFRecordDataset(path)`
I1010 04:55:43.283555 140631071238016 calc_metrics_utils.py:414] denotation_accuracy=0.749673
dev denotation accuracy: 0.7497
I1010 04:55:43.428153 140631071238016 run_task_main.py:152] dev denotation accuracy: 0.7497
I1010 04:56:35.534448 140631071238016 calc_metrics_utils.py:414] denotation_accuracy=0.736554
test denotation accuracy: 0.7366
I1010 04:56:35.813148 140631071238016 run_task_main.py:152] test denotation accuracy: 0.7366
```

Were you able to reproduce the accuracies exactly?
